pagopa/io-app
# chore(IDPay): [IODPAY-90] Add `IdPayConfig` remote feature flag

## Short description
This PR introduces the remote ff to enable and disable the IDPay app features.

## List of changes proposed in this pull request
- Added `isIdPayEnabledSelector`
- Replaced local flag with remote flag selector where possible

## How to test
Using the dev-api-server try to change min_app_version of `idPay` remote config and check if the IDPay features in the app are available/displayed.
